### PR TITLE
Create Disclaimer component

### PR DESCRIPTION
**Pull request by grosbouddha**
TESTED=Unit tests and locally in Storybook

image

### CODE REVIEW

**Comment by pith** [comment]
The spacing between the icon and the text is a bit big.

File: src/disclaimer/specifications/disclaimer.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+
+## Disclaimer
+
+```js
+import { PhoneField } from '@blablacar/ui-library'
```
Comment (Tuxosaurus) [inline src/disclaimer/specifications/disclaimer.md]: not PhoneField :)

Comment (grosbouddha) [reply src/disclaimer/specifications/disclaimer.md]: Thanks :)

**Comment by grosbouddha** [comment]
> Nice. But the spacing between the icon and the text seems a bit big on your screenshot.

@pith: The spacing is standard, I used a generic Pixar item.

**Pull request merged**